ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -762,7 +767,7 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
     free(messageMemory);
 }
 
-static PrjFS_Result HandleEnumerateDirectoryRequest(const MessageHeader* request, const char* relativePath)
+static PrjFS_Result HandleEnumerateDirectoryRequest(const MessageHeader* request, const char* fullPath, const char* relativePath)
 {
 #ifdef DEBUG
     cout
```

Comment:
As part of this debug, would we want to print out the new full Path?  This view cuts it off but you can see the debug on the full file.  This question for all the affected functions.